README.md

```diff
@@ -421,6 +423,9 @@ following properties:
 
     * `reason` (String) -- A string describing the reason for an unsupported video recommendation.
       For example, `'No camera was found.'`
+    
+    * `qualityLimitationReason` (String) --  The qualityLimitationReason indicates the reason behind 
+       any limitations in media quality during a session. It can have values: `cpu` for CPU overload, `bandwidth` for insufficient network bandwidth and `none` if there is no limition.
```

Comment:
When I run the sample app, `results.video.qualityLimitationReason` is undefined.

Reply:
Hey @jeffswartz, could you please check if you're using the latest version? I just tested it from scratch, and it's works

Reply:
@vona-ben The issue occurs in Node 16. The `file://..` reference in the sample/package.json file is invalid (in Node 16 only, it seems), and it appears that when you run `npm install` (in the sample app), it pulls the opentok-network-test-js library from npmjs! I added a commit to fix this.